client/src/www/views/root_view/privacy_acknowledgement_dialog/index.ts

```diff
@@ -97,15 +108,9 @@ export class PrivacyAcknowledgementDialog extends LitElement {
           <p>${this.localize('privacy-text')}</p>
         </article>
         <fieldset slot="actions">
-          <md-text-button
-            @click="${() =>
-              window.open(
-                'https://support.getoutline.org/s/article/Data-collection',
-                '_blank'
-              )}"
-          >
+          <a href="https://support.getoutline.org/s/article/Data-collection">
```

Comment:
Why these changes to the dialogs? They seem unrelated to the header and navigation, but maybe I'm missing something.

Reply:
Yeah if you don't mind putting unrelated changes into their own PR.